scripts/check_maintainers.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2024 Vestas Wind Systems A/S
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import sys
+
+from github_helpers import get_github_object
+from get_maintainer import Maintainers
+from github.GithubException import UnknownObjectException
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        description=__doc__, allow_abbrev=False)
+
+    parser.add_argument(
+        "-m", "--maintainers",
+        metavar="MAINTAINERS_FILE",
+        help="Maintainers file to load. If not specified, MAINTAINERS.yml in "
+             "the top-level repository directory is used, and must exist. "
+             "Paths in the maintainers file will always be taken as relative "
+             "to the top-level directory.")
+
+    return parser.parse_args()
+
+def main() -> None:
+    args = parse_args()
+    maintainers = Maintainers(args.maintainers)
+    gh = get_github_object()
+    gh_users = []
+    notfound = []
+
+    for area in maintainers.areas.values():
+        gh_users = list(set(gh_users + area.maintainers + area.collaborators))
+
+    gh_users.sort()
+
+    print('Checking maintainer and collaborator user accounts on GitHub:')
+    for gh_user in gh_users:
+        try:
+            print('.', end='', flush=True)
+            gh.get_user(gh_user)
+        except UnknownObjectException:
+            notfound.append(gh_user)
+    print('\n')
+
+    if notfound:
+        print('The following GitHub user accounts do not exist:')
+        print('\n'.join(notfound))
+        sys.exit(1)
+    else:
+        print('No non-existing user accounts found')
```

Comment:
That is the very rare case of github account disappearing and that is good, however, it makes sense to add a check if entries in the maintainer file are part of the zephyr project, if someone is in the file and not part of the project, nothing get assgined, so it is as if the account does not exist on github...

Reply:
Good point. I can extend the script functionality to cover this as well.

Reply:
I've updated the script to check for non-collaborator entries as well - how should we handle these?